### ISSUE TITLE
[Idea/Add] get recommended Java Version from the api/v2

### ISSUE THREAD

**Issue by LuckCrafter** [PaperMC/bibliothek#77]
I'll like to see a feature to get the recommended Java version for PaperMC (something like in the [docs](https://paper.readthedocs.io/en/latest/server/getting-started.html#requirements))

**Comment by electronicboy** [comment]
recommended java version just seems completely irrelevant to the distribution system

**Comment by MiniDigger** [comment]
I agree with cat, but I also see the other side, the api is our main communication channel to hosts, such a feature could greatly reduce the work they have to do to deploy stuff.
maybe we can add a tagging system so we can add metadata like this to version groups?

**Comment by electronicboy** [comment]
My issue is, what constitutes as recommended? Tagging the build version is trivial, but, the notation of recommendations is somewhat iffy as we always just recommend the latest version that we know works for X a release, and am not sure that's something we should particularly be tryna maintain here

**Comment by MiniDigger** [comment]
we already maintain it on the docs, as OP said, so idk,. what do others think?

**Comment by Machine-Maker** [comment]
Why not minimum java version instead of recommended?

**Comment by sulu5890** [comment]
Minimum you can just get via the class file version, recommended (especially back on 1.16 and below) was harder and just involved hard coding a table. Also see https://github.com/PaperMC/Paper/issues/5630 for another possible approach/more rational.

**Comment by electronicboy** [comment]
We maintain it on the docs in a place which isn't updating the metadata of several dozen builds at once? Unless you mean per release in which kills the granularity and creates some complexities when java issues are fixed or the max supported ver is bumped, etc
I just feel that a recommended version is far too nuanced for something which others will push onto others less fortunate

**Comment by MiniDigger** [comment]
I would have defined it on the version group
